genai-perf/genai_perf/inputs/converters/tensorrtllm_engine_converter.py

```diff
@@ -80,6 +83,8 @@ def _add_request_params(self, payload: Dict, config: InputsConfig) -> None:
             payload["request_output_len"] = [num_tokens]
             if config.output_tokens_deterministic:
                 payload["min_length"] = [num_tokens]
+        if config.set_end_id:
+            payload["end_id"] = [config.tokenizer._tokenizer.eos_token_id]
```

Comment:
I am a bit concerned about adding a new CLI option for this specific use cases. Since we try to avoid adding too much options to the tool, and this can be achieved through the `--extra-inputs <name>:<value>` as well.
cc @dyastremsky

Reply:
Yeah if this can be done via [--extra-inputs](https://github.com/triton-inference-server/perf_analyzer/tree/main/genai-perf#--extra-inputs-str), let's skip this change.

Reply:
It is moved to --extra-inputs. however, code change is still necessary unless use directly provide eos token id (instead of fetching from tokenizer). Please let me know if current approach looks good. Thanks.